src/herder/test/HerderTests.cpp

```diff
@@ -3560,105 +3560,119 @@ TEST_CASE("soroban txs each parameter surge priced", "[soroban][herder]")
     auto networkID = sha256(getTestConfig().NETWORK_PASSPHRASE);
     uint32_t baseTxRate = 1;
     uint32_t numAccounts = 100;
-    auto test =
-        [&](std::function<void(SorobanNetworkConfig & cfg)> tweakConfig) {
-            auto simulation = Topologies::core(
-                4, 1, Simulation::OVER_LOOPBACK, networkID, [&](int i) {
-                    auto cfg = getTestConfig(i, Config::TESTDB_ON_DISK_SQLITE);
-                    cfg.TESTING_UPGRADE_MAX_TX_SET_SIZE = 100;
-                    return cfg;
+    auto test = [&](std::function<void(SorobanNetworkConfig & cfg)> tweakConfig,
```

Comment:
Nit: maybe change `tweakConfig` to `tweakSorobanConfig` or `tweakNetworkConfig` for clarity.